mono/mini/aot-compiler.c

```diff
@@ -7936,6 +7939,101 @@ mono_aot_split_options (const char *aot_options)
 	return args;
 }
 
+static gboolean
+parse_cpu_features (MonoCPUFeatures *cpu_features_enabled, MonoCPUFeatures *cpu_features_disabled, const gchar *str)
+{
+	*cpu_features_enabled = (MonoCPUFeatures) 0;
+	*cpu_features_disabled = (MonoCPUFeatures) 0;
+
+	gchar **attrs = g_strsplit (str, ";", -1);
+
```

Comment:
Semicolons are not good separators since they also used by the shell. Id suggest simply using multiple -mattr arguments if needed.

Reply:
Done, now the syntax is `mattr=+avx2,mattr=-lzcnt,mattr=+bmi2`